### PR TITLE
removed heka, moved supervisord

### LINKED ISSUE #165: hekad: ERROR (not running) when deploying a new stack
Deploying a new stack and getting this all the time:
```
NOTIFIED: [common | restart heka] ********************************************* 
failed: [ec2-52-25-78-92.us-west-2.compute.amazonaws.com] => {"failed": true}
msg: hekad: ERROR (not running)
hekad: ERROR (abnormal termination)


FATAL: all hosts have already failed -- aborting

TASK: [oauth | configure nginx] *********************************************** 
FATAL: no hosts matched or all hosts have already failed -- aborting

```

### PR DESCRIPTION
**Pull request by dannycoates**
@vladikoff @jrgm fixes #165 #160

### CODE REVIEW

**Comment by vladikoff** [comment]
trying that now, deploying a "vlad.dev.lcip.org" stack.

Thanks @dannycoates !

**Comment by dannycoates** [comment]
shoot. this didn't get the supervisor change...

**Comment by vladikoff** [comment]
(/¯◡ ‿ ◡)/¯ ~ ┻━┻

**Comment by vladikoff** [comment]
Yeah just got 
```
TASK: [common | start supervisord] ******************************************** 
failed: [ec2-54-69-13-8.us-west-2.compute.amazonaws.com] => {"failed": true}
msg: start: Job failed to start


FATAL: all hosts have already failed -- aborting

```

**Comment by dannycoates** [comment]
updated

**Comment by vladikoff** [comment]
oops posted a comment in the wrong bug: https://github.com/mozilla/fxa-dev/issues/165#issuecomment-122977583

**Comment by vladikoff** [comment]
```
TASK: [common | link supervisorctl] ******************************************* 
failed: [ec2-52-26-241-62.us-west-2.compute.amazonaws.com] => {"changed": true, "cmd": ["ln", "-s", "/usr/local/bin/supervisorctl", "/usr/bin/supervisorctl"], "delta": "0:00:00.044867", "end": "2015-07-20 19:40:52.483864", "rc": 1, "start": "2015-07-20 19:40:52.438997", "warnings": ["Consider using file module with state=link rather than running ln"]}
stderr: ln: failed to create symbolic link '/usr/bin/supervisorctl': File exists

FATAL: all hosts have already failed -- aborting
```

File: roles/common/tasks/main.yml [hunk 1]
```diff
@@ -77,22 +77,9 @@
...
-- name: supervise heka
-  sudo: true
-  copy: src=hekad.conf dest=/etc/supervisor.d/hekad.conf
-  notify: update supervisor
-
-- name: configure heka
-  sudo: true
-  copy: src=hekad.toml dest=/etc/heka.d/!hekad.toml
-  notify: restart heka
+  command: ln -s /usr/local/bin/supervisorctl /usr/bin/supervisorctl
```
Comment (vladikoff) [inline roles/common/tasks/main.yml]: should this be 
```
command: ln -fs
```
otherwise it will fail once you run `make TARGET` more than once?

Comment (dannycoates) [inline roles/common/tasks/main.yml]: yes

**Comment by dannycoates** [comment]
@vladikoff 💩 sorry, let me regroup and fix this for real.

**Comment by vladikoff** [comment]
@dannycoates should we take out Kibana as well? How is that used?

**Comment by dannycoates** [comment]
ok, this works now, and i actually ran it :)

**Pull request merged**